### PR TITLE
I18n support

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

**Comment by alebinson** [comment]
@asafMasa this is a final one, even with newly published dependent NPM packages

File: src/common/i18n/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { MessageFormatElement } from 'intl-messageformat-parser';
+
+import messages_en from "./en.json";
```
Comment (asafMasa) [inline src/common/i18n/index.ts:3]: run lint on this file (" and ' in the same file)

**Review by asafMasa — CHANGES_REQUESTED** [review]
why do we have conflicts directory in our app? please find and remove this name

File: src/common/i18n/index.ts (hunk 1 above)
Comment (alebinson) [reply src/common/i18n/index.ts:3]: actually lint was run, nothing about it, strange.
Fixed

File: src/common/i18n/en.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "polygon-selection.clear-menu_option.text": "Clear",
+
+    "custom-bbox.dialog.title": "Define bbox coordinates",
+    "custom-bbox.dialog-field.top_right_lat.text": "Top Right Lat",
+    "custom-bbox.dialog-field.top_right_lon.text": "Top Right Lon",
+    "custom-bbox.dialog-field.bottom_left_lat.text": "Bottom Left Lat",
+    "custom-bbox.dialog-field.bottom_left_lon.text": "Bottom Left Lon",
+    "custom-bbox.form-error.x-distance.text": "X distance is exceeded the limit",
+    "custom-bbox.form-error.y-distance.text": "Y distance is exceeded the limit"
+}
```
[the remark this answers is not in the document]
Comment (alebinson) [reply src/common/i18n/en.json:21]: done

File: src/common/i18n/he.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "custom-bbox.dialog.title": "הגדר קוורדינתות אזור היצוא",
+    "custom-bbox.dialog-field.top_right_lat.text": "רוחב נק' ימין עליונה",
+    "custom-bbox.dialog-field.top_right_lon.text": "ארך נק' ימין עליונה",
+    "custom-bbox.dialog-field.bottom_left_lat.text": "רוחב נק' שמאל תחתונה",
+    "custom-bbox.dialog-field.bottom_left_lon.text": "ארך נק' שמאל תחתונה",
+    "custom-bbox.form-error.x-distance.text": "ארך מעל המותר",
+    "custom-bbox.form-error.y-distance.text": "רוחב מעל המותר"
+
+
+}
```
[the remark this answers is not in the document]
Comment (alebinson) [reply src/common/i18n/he.json:23]: done

File: src/App.tsx [hunk 4]
```diff
@@ -16,22 +17,30 @@ import '@map-colonies/react-core/dist/dialog/styles';
 import '@map-colonies/react-core/dist/textfield/styles';
 
 import ExporterView from './conflicts/views/exporter-view';
+import MESSAGES from './common/i18n';
 
 const App: React.FC = () => {
   const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
+  const [lang, setLang] = useState('he');
```
[the remark this answers is not in the document]
Comment (alebinson) [reply src/App.tsx:24]: done

**Comment by alebinson** [comment]
**conflicts** will be removed later, first want to work on functionality, to deliver on time.

**Review by asafMasa — APPROVED** [review]

**Pull request merged**